src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -18,6 +20,7 @@
    :witan/param-schema {:last-proj-year s/Int}
    :witan/exported? true}
   [{:keys [loop-year]} {:keys [last-proj-year]}]
+  (doseq [y [loop-year last-proj-year]] (utils/property-holds? y m-utils/year? (str y " is not a year")) )
```

Comment:
As `loop-year` and `last-proj-year` both come directly from the inputs and params, this check _should_ be at the schema level. For example, `:witan/param-schema {:last-proj-year (s/constrained s/Int m-utils/year?)} ... `